certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -2461,6 +2461,315 @@
           }
```

Comment:
@anfuca Your PR contains other connector file changes which is not allowed. Please remove them from your PR. Please raise one PR for each connector.